python/modules/IcePy/ValueFactoryManager.cpp

```diff
@@ -127,20 +118,19 @@ IcePy::ValueFactoryManager::findValueFactory(string_view id) const
 PyObject*
 IcePy::ValueFactoryManager::getObject() const
```

Comment:
We have multiple getObject methods, in the extension and some return a strong reference others a borrow reference. I want to cleanup this in a follow-up PR and always return a borrow reference.